hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleUtility.java

```diff
@@ -318,8 +312,9 @@ public static String calculateStringHash(final @NonNull Schedule scheduleToHash)
         if (scheduleToHash.scheduledTransaction() != null) {
             addToHash(hasher, scheduleToHash.scheduledTransaction());
         }
-        if (scheduleToHash.expirationTimeProvided() != null) {
-            addToHash(hasher, scheduleToHash.expirationTimeProvided());
+        if (scheduleToHash.providedExpirationSeconds() != Schedule.DEFAULT.providedExpirationSeconds()) {
+            hasher.putLong(scheduleToHash.providedExpirationSeconds());
+            //            addToHash(hasher, scheduleToHash.providedExpirationSeconds());
```

Comment:
intentional comment?

Reply:
Yeah, this is related to Joseph's request to change this line. Changing it causes a test to break. I pushed the change so that he could see what I did. He's now planning to fix it in a separate PR. I added a @todo tag for it.